systemd/systemd.go

```diff
@@ -326,6 +326,14 @@ const (
 	EmulationModeBackend
 )
 
+type MountUpdateStatus int
+
+const (
+	MountUnchanged MountUpdateStatus = iota
+	MountUpdated
+	MountCreated
+)
+
```

Comment:
not sure this needs to exported, they seem to be used only internally

Reply:
Done.